### ISSUE TITLE
[Feature Request]: Change the definition of `BIT7Z_STRING`

### ISSUE THREAD

**Issue by 836454543** [rikyoz/bit7z#167]
### Feature description

If define `BIT7Z_USE_NATIVE_STRING`, the definition of `BIT7Z_STRING` is `L##str`.
This can be problematic in some cases.
```cpp
/* define a macro called `STR` */
#define STR "hello"

/* use `BIT7Z_STRING` with `STR` */
BIT7Z_STRING(STR);  /* equal to `LSTR`, however `L"hello"` is expected */
```
The following is a better way to define `BIT7Z_STRING`
```cpp
#define _BIT7Z_STRING(str) L##str
#define BIT7Z_STRING(str) _BIT7Z_STRING(str)
```

### Code of Conduct

- [X] By submitting this issue, I agree to follow bit7z's [Code of Conduct](https://github.com/rikyoz/bit7z/blob/master/CODE_OF_CONDUCT.md)

**Comment by rikyoz** [comment]
Hi!
I think it might be helpful for the users of bit7z, so I've just pushed a commit to the branch `release/v4.0.0` that changes the definition of `BIT7Z_STRING` as you suggested.

Thanks for the suggestion!

**Comment by rikyoz** [comment]
Implemented in [v4.0.0](https://github.com/rikyoz/bit7z/releases/tag/v4.0.0).
